app/views/merchant/orders/show.html.erb

```diff
@@ -28,6 +28,7 @@
       <th>Price</th>
       <th>Quantity</th>
       <th>Subtotal</th>
+      <th>Fulfillement</th>
```

Comment:
Had to add this column to the table for the iteration below to have a place to populate

Reply:
BANG BANG!